microservices/front-end/src/components/chat/Chat.tsx

```diff
@@ -83,15 +103,21 @@ const Chat: FC<ChatProps> = ({ socket }) => {
     <div style={{ position: "relative", height: "100%" }}>
       <MainContainer>
         <ChatContainer>
-          {/* <ConversationHeader>
-            <ConversationHeader.Content userName="Emily" /> // get display name
-            on initial setup
-          </ConversationHeader> */}
-          <MessageList>{...messages}</MessageList>
+          <MessageList
+            typingIndicator={
+              otherTyping && (
+                <TypingIndicator content="other person is typing" />
```

Comment:
Is it possible to specify the username in the typing indicator (e.g., `"john is typing..."` or `"Henry is typing..."`)?